.editorconfig

```diff
@@ -7,3 +7,6 @@ charset = utf-8
 end_of_line = lf
 insert_final_newline = true
 trim_trailing_whitespace = true
+
+[*_test.go]
+trim_trailing_whitespace = false
```

Comment:
I don't see any trailing whitespace in the `key_test.go`, how is this making a difference?